volsupervisor/loop.go

```diff
@@ -16,6 +19,33 @@ var (
 	volumeMutex = &sync.Mutex{}
 )
 
+func (dc *DaemonConfig) updateVolumes() {
+	myVolumes, err := dc.Config.ListAllVolumes()
+	fmt.Println(myVolumes)
```

Comment:
Ahh, you later removed it in this commit anyways:  https://github.com/contiv/volplugin/pull/435/commits/1343d7c5366f85dc5e46909e9246681d7b157573